
class OutOfDomainException(Exception):
    """
    Исключение, вызываемое при выходе из области определения функции.
    """
    pass